deepchem/models/torch_models/tests/test_chemberta.py

```diff
@@ -0,0 +1,125 @@
+
+    # test multitask regression
+    model = Chemberta(task='finetuning',
+                      tokenizer_path=tokenizer_path,
+                      mode='regression',
+                      n_tasks=2)
+    loss = model.fit(smiles_multitask_regression_dataset, nb_epoch=1)
+    assert loss
+
+    # test classification
+    y = np.random.choice([0, 1], size=smiles_regression_dataset.y.shape)
+    dataset = dc.data.NumpyDataset(X=smiles_regression_dataset.X,
+                                   y=y,
+                                   w=smiles_regression_dataset.w,
+                                   ids=smiles_regression_dataset.ids)
+    model = Chemberta(task='finetuning',
+                      tokenizer_path=tokenizer_path,
+                      mode='classification')
+    loss = model.fit(dataset, nb_epoch=1)
+    assert loss
+
+
+@pytest.mark.torch
+def test_chemberta_load_from_pretrained(tmpdir, smiles_regression_dataset):
+    pretrain_model_dir = os.path.join(tmpdir, 'pretrain')
+    finetune_model_dir = os.path.join(tmpdir, 'finetune')
+    tokenizer_path = 'seyonec/PubChem10M_SMILES_BPE_60k'
+    pretrain_model = Chemberta(task='pretraining',
+                               tokenizer_path=tokenizer_path,
+                               mode='mlm',
+                               model_dir=pretrain_model_dir)
+    pretrain_model.save_checkpoint()
+
+    finetune_model = Chemberta(task='finetuning',
+                               tokenizer_path=tokenizer_path,
+                               mode='regression',
+                               model_dir=finetune_model_dir)
+    finetune_model.load_from_pretrained(pretrain_model_dir)
+
+    # check weights match
+    pretrain_model_state_dict = pretrain_model.model.state_dict()
+    finetune_model_state_dict = finetune_model.model.state_dict()
+
+    pretrain_base_model_keys = [
+        key for key in pretrain_model_state_dict.keys() if 'roberta' in key
+    ]
+    matches = [
+        torch.allclose(pretrain_model_state_dict[key],
+                       finetune_model_state_dict[key])
+        for key in pretrain_base_model_keys
+    ]
+
+    assert all(matches)
+
+
+@pytest.mark.torch
+def test_chemberta_save_reload(tmpdir):
+    tokenizer_path = 'seyonec/PubChem10M_SMILES_BPE_60k'
+    model = Chemberta(task='finetuning',
+                      tokenizer_path=tokenizer_path,
+                      mode='regression',
+                      model_dir=tmpdir)
+    model._ensure_built()
+    model.save_checkpoint()
+
+    model_new = Chemberta(task='finetuning',
+                          tokenizer_path=tokenizer_path,
+                          mode='regression',
+                          model_dir=tmpdir)
+    model_new.restore()
+
+    old_state = model.model.state_dict()
+    new_state = model_new.model.state_dict()
+    matches = [
+        torch.allclose(old_state[key], new_state[key])
+        for key in old_state.keys()
+    ]
+
+    # all keys values should match
```

Comment:
Can we add a unit test checking that we can load the public chemberta weights (assuming this test isn't too slow)

Reply:
added test in #3390. Since this class reuses implementation from base model, I think a test in base class will be sufficient.